dev/io.openliberty.data.internal_fat/test-applications/DataTestApp/src/test/jakarta/data/web/DataTestServlet.java

```diff
@@ -4968,7 +4968,7 @@ public void testRepositoryUpdateMethodsMultiplyAndDivide() {
         assertEquals(19.5f, p.height, 0.01f);
 
         // divide width and append to description via query by method name
-        assertEquals(true, packages.updateByIdDivideWidthAddDescription(990003, 2, " halved"));
+        assertEquals(true, packages.reduceWidthAppendDescription(990003, 2, " halved"));
 
         p = packages.findById(990003).orElseThrow();
         assertEquals(11.4f, p.length, 0.01f);
```

Comment:
Did you want to change the name of the call to `updateByVinIdAddPrice` to not use `updateBy` on lines:
```
3626
3645
5266
```

Reply:
No, that is the one where I intentionally left `updateBy` in the name, but letting `@Query` define the meaning instead.